centinel/vpn/cli.py

```diff
@@ -213,6 +214,25 @@ def create_config_files(directory):
         configuration.write_out_config(conf_file)
 
 
+def experiments_available(config):
+    logging.info("Starting to check for experiments with %s",
+                 config['server']['server_url'])
+    try:
+        user = centinel.backend.User(config)
```

Comment:
You also need to pull down the results from the user. This doesn't actually pull down the experiments or check the experiments list, it just creates a user if one doesn't already exist.